apps/i18n/spritelab/en_us.json

```diff
@@ -0,0 +1,35 @@
+{
+  "animationPicker_drawYourOwn": "Draw your own",
+  "animationPicker_error": "Error: {message}",
+  "animationPicker_failedToParseImage": "The image could not be parsed",
+  "animationPicker_title": "Animation Library",
+  "animationPicker_unsupportedType": "Sorry, this file type is not supported.",
+  "animationPicker_unsupportedSize": "Please make sure the image you are trying to upload is smaller than 100 KB.",
+  "animationPicker_uploadImage": "Upload image",
+  "animationPicker_uploading": "Uploading...",
+  "animationPicker_warning": "Warning: Do not upload anything that contains personal information.",
```

Comment:
These strings could potentially be moved to [apps/i18n/common/en_us.json](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/i18n/common/en_us.json) since they're now shared between multiple apps.

Reply:
I'll do this as a follow up!